### PR TITLE
Added MIT License

### LINKED ISSUE #73: [Docs] Add MIT License
**Current State:**
LICENSE IS EMPTY
**Desired State:**
ADD MIT LICENSE

### PR DESCRIPTION
**Pull request by san-coding**
**Description**

This PR fixes #73 

Added MIT License 
**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudhay23** [comment]
Great

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Sudhay Senthilkumar
```
Comment (sudhay23) [inline LICENSE:3]: This can be SS Ventures ig lol

**Review by sudhay23 — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (san-coding) [reply LICENSE]: Have made the requested changes

**Review by sudhay23 — APPROVED** [review]
Good job

**Pull request merged**